contracts/LoanManager.sol

```diff
@@ -0,0 +1,49 @@
+pragma solidity 0.4.15;
+
+contract LoanManager {
+
+    address public owner = msg.sender;
+
+    mapping (address => uint) public loans;
+
+    function borrow(uint amount) public returns (bool) {
+        loans[msg.sender] = add(loans[msg.sender], amount);
+        return true;
+    }
+
+
+    function repay(address borrower, uint amount) public returns(bool, string){
+        if (msg.sender != owner){
+            return (false, "You are not the loan owner");
+        } else if (loans[borrower] < amount){
```

Comment:
Do you cover it in your tests?

Reply:
Test: "should not allow not owner to repay"